### PR TITLE
Bug 1224063 - Refactor System to use modern L10n API, part 1

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1224063

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7cf9e33686ce3cde5c981d7b67a796177fb1a6c6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6fbc75513c4851057bb20ff743b10d293af1c338

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3b8e1b60f7fd7cdf554793c76158364df0d5b39f

File: apps/system/test/unit/app_install_manager_test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
           MockApp, MockApplications, MockChromeEvent, MockNavigatormozApps,
           MockNavigatorWakeLock, MocksHelper, MockL10n, MockModalDialog,
           MockNotificationScreen, MockService,
-          MockSystemBanner, MockUtilityTray */
+          MockSystemBanner, MockUtilityTray, l10nAssert */
```
Comment (etiennesegonzac) [inline apps/system/test/unit/app_install_manager_test.js]: kewl

**Pull request merged**